cmd/loadtest/loadtest.go

```diff
@@ -918,6 +930,34 @@ func loadtestCall(ctx context.Context, c *ethclient.Client, nonce uint64, ltCont
 	t2 = time.Now()
 	return
 }
+
+func loadtestCallPrecompiledContracts(ctx context.Context, c *ethclient.Client, nonce uint64, ltContract *contracts.LoadTester, useSelectedAddress bool) (t1 time.Time, t2 time.Time, err error) {
+	var f int
+	ltp := inputLoadTestParams
+
+	chainID := new(big.Int).SetUint64(*ltp.ChainID)
+	privateKey := ltp.ECDSAPrivateKey
+	iterations := ltp.Iterations
+	if useSelectedAddress {
+		f = int(*ltp.Function)
```

Comment:
Is the idea here that you would pass `9` in order to call `blake2f`

Reply:
That's correct, `ltp.Function` is multipurpose for an opcode and a precompiled address.